styles/styles.css

```
@@ -0,0 +1,243 @@
+@font-face {
+    font-family:"Rubik" ;
+    font-style: normal;
+    font-weight:400 ;
+    src: url("../fonts/rubik/rubik-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family:"Rubik" ;
+    font-style: normal;
+    font-weight:700 ;
+    src: url("../fonts/rubik/rubik-700.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family:"Raleway" ;
+    font-style: normal;
+    font-weight:800 ;
+    src: url("../fonts/raleway/raleway-800.woff2") format("woff2");
+    font-display: swap;
+}
+
+body {
+    font-family: "Rubik";
+    font-style: normal;
+    font-size:18px ;
+    line-height:30px ;
+    background-color:#F0F0F0;
+    color: black;
+}
+
+.page-header {
+    background-color: #ffe17f;
+    color: black;
+}
+.search {
+    font-size: 16px;
+    line-height: 20px;
+}
+
+.main-index {
+    color:black;
+    background-color:white;
```

```suggestion
    color: #000000;
    background-color: #ffffff;
```
у тебя сейчас намешано. где-то цвет словом, а где-то через хекс. Должно быть единообразно. Сделай везде через хекс